### PR TITLE
[MIG] account_invoice_export: Migration to 14.0

### PR DESCRIPTION
**Pull request by phuctranfxvn**
Requires OCA modules:
- `account_invoice_transmit_method` which is being migrated in https://github.com/OCA/account-invoicing/pull/823

### CODE REVIEW

**Comment by yvaucher** [comment]
:warning: there are some on progress changes in #213

File: account_invoice_export/__manifest__.py [hunk 1]
```diff
@@ -3,15 +3,16 @@
...
     "name": "Account Invoice Export",
-    "version": "13.0.1.0.0",
+    "version": "14.0.1.0.0",
     "category": "Invoicing Management",
     "license": "AGPL-3",
     "summary": "",
     "author": "Camptocamp SA, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/edi",
-    "depends": ["account_invoice_transmit_method", "queue_job"],
+    "depends": ["sale", "account_invoice_transmit_method", "queue_job"],
```
Comment (TDu) [inline account_invoice_export/__manifest__.py]: Is the `sale` module addition to the dependencies needed ?

File: account_invoice_export/models/account_move.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 from odoo import _, fields, models
 from odoo.exceptions import UserError, except_orm
 
-from odoo.addons.queue_job.job import job
+# from odoo.addons.queue_job.job import job
```
Comment (TDu) [inline account_invoice_export/models/account_move.py]: Should be deleted if not needed anymore

**Comment by TDu** [comment]
Also in the commit related to the pre-commit hooks "account_invoice_export: black, isort, prettier" are all changes generated by the hooks ?
Because this one seems a cosmetic one 
https://github.com/OCA/edi/pull/299/commits/63eb325334b0c0fa63246dee17aa909809ff6714#diff-bcb6b814ad9e78b28e5c3b4d0958cb1c7e257d2ddebb48e7169fd3e6a929caffR24
And the previous maybe as well ?

File: account_invoice_export/models/account_move.py (hunk 2 above)
Comment (phuctranfxvn) [reply account_invoice_export/models/account_move.py]: fixed, thanks

File: account_invoice_export/__manifest__.py (hunk 1 above)
Comment (phuctranfxvn) [reply account_invoice_export/__manifest__.py]: ah it was a mistake, fixed, thanks

**Comment by TDu** [comment]
LG, there is just one more thing for me is this later commit https://github.com/OCA/edi/commit/51b5aed2c5f58ee0fd6cce00b9eb72cecf6c5e0a#diff-8abaea68b746a4f70ee3b565778ece57657b089f5751cd9c41f66c8b5b18feb3 also need to be included. Thank you

**Comment by hailangvn** [comment]
@TDu, The commit has been included. Thanks for reviewing.

**Review by TDu — APPROVED** [review]
Thank you for the migration

File: test-requirements.txt [hunk 3]
```diff
@@ -1,3 +1,4 @@
 freezegun
 odoo_test_helper
 responses
+odoo14-addon-account-invoice-transmit-method @git+https://github.com/akretion/account-invoicing@14-mig-transmit_method#subdirectory=setup/account_invoice_transmit_method
```
Comment (mmequignon) [inline test-requirements.txt]: Is it a common practice to target a module that is not in an OCA repo ?

Comment (hailangvn) [reply test-requirements.txt]: Thanks @mmequignon for your comment. We are working on this.

Comment (hailangvn) [reply test-requirements.txt]: Thanks @mmequignon for the comment. That dependency has been merged by https://github.com/OCA/account-invoicing/pull/823. So the `test-requirements.txt` is updated just now.

**Review by mmequignon — APPROVED** [review]

**Pull request merged**